src/Configurations/AccessRules.php

```diff
@@ -10,6 +10,11 @@ public function setIP(string $value)
     {
         $this->config = ['target' => 'ip', 'value' => $value];
```

Comment:
let's just make this method cover both use cases for simplicity sake

```suggestion
        if (strpos($value, ':') !== false) {
          $this->config = ['target' => 'ip6', 'value' => $value];
        } else {
          $this->config = ['target' => 'ip', 'value' => $value];
        }
```